recipes/powerplantmatching/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "powerplantmatching" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "f84b8b715e00a6bd74c340a04f71bbd00c6e959916e4b9d85e6450358cf2f84e" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - numpy
+    - scipy
+    - pandas>=0.23.0
+    - networkx>=1.10
+    - pycountry
+    - xlrd
+    - seaborn
+    - pyyaml
+    - requests
+    - matplotlib
+    - geopy
+    - pip:
+      - entsoe-py
```

Comment:
This should be packaged on conda-forge first

Reply:
Thank you for reviewing, then I will set up a conda-forge package for entsoe-py first